convert2rhel/utils.py

```diff
@@ -618,4 +625,44 @@ def set_locale():
     os.environ.update({"LC_ALL": "C", "LANG": "C"})
 
 
+def get_secrets_from_conf_file(path_to_cfg=None):
+    """Get the secret (activation key or password) from config file.
+
+    The order is:
+    1. if filename is set then load from it
+    2. ~/.convert2rhel.ini
+    3. /etc/convert2rhel.ini
+
+    Return {"password": <value>, "activation-key": <value>}.
+    If the secret isn't set, default value is None.
+    """
+    header = "SECRET"  # header in config file
+    secret_names = ("password", "activation-key")
+    config = configparser.ConfigParser()
+    paths = CONFIG_PATHS
```

Comment:
I think this global constant is not very useful right here, what do you think to placing it inside the function instead? 

```suggestion
    paths = ["~/convert2rhel.ini", "/etc/convert2rhel.ini"]
```

Reply:
That was a variant. But I choose this one, because I think that's improve readability of the code - you see the default paths in the beginning of the file without digging to it. And it's easy to add a new one in case somebody needs that.